packages/ui/views/app/index.tsx

```diff
@@ -59,7 +60,11 @@ export default class AppView extends Component<Props, EngineState> {
     } = this.state
 
     if (isInitializing) {
-      return <Logo />
+      return <ThemeImage name={'logo'} />
```

Comment:
When passing a string, `{` and `}` are not required. Just use `<ThemeImage name='logo' />`